ptf/saisanity.py

```diff
@@ -0,0 +1,514 @@
+
+from __future__ import print_function
+
+from sai_thrift.sai_headers import *
+
+from ptf.testutils import *
+from ptf.dataplane import DataPlane
+
+from sai_base_test import *
+
+mac1 = '00:11:11:11:11:11'
+mac2 = '00:22:22:22:22:22'
+mac3 = '00:33:33:33:33:33'
+mac4 = '00:12:12:12:12:13'
+
+
+class L2TrunkToTrunkVlan(PlatformSaiHelper):
+
+    def remove_bridge_port(self):
+        for index in range(0, len(self.port_list)):
+            port_bp = getattr(self, 'port%s_bp' % index)
+            sai_thrift_remove_bridge_port(self.client, port_bp)
+
+
+    def create_bridge_ports(self):
```

Comment:
no, in this case, it will not conflict. 
indeed we create the bp in SaiHelper. (that's why reset bridge need happed before create it). 
In this sanity test, it inherits from SaiHelperBase, so there is not any bp created yet.